apps/backend/src/main.ts

```diff
@@ -1,21 +1,25 @@
-import { PrismaClient, type User } from '../generated/prisma/index.js';
 import express from 'express';
+import dotenv from 'dotenv';
+import { prisma } from './client.js';
+import { usersRouter } from './routes/users.routes.js';
 
-const prisma = new PrismaClient();
+dotenv.config();
 
 async function main() {
-  const allUsers: User[] = await prisma.user.findMany();
-  console.log(allUsers);
-
   const app = express();
-  const port = 5000;
+  const PORT = process.env.PORT || 3000;
```

Comment:
```suggestion
  const PORT = process.env.PORT ?? 3000;
```

https://typescript-eslint.io/rules/prefer-nullish-coalescing/